/**:
  ros__parameters:
    visualize: true

    # when enabled,finds point clouds with most overlap automatically
    # this method is more general and might be more precise but is slower
    use_fitness_based_calibration: false

    # Topics for all lidars
    lidar_topics: [lidar_1, lidar_2, lidar_3]

    # read initial_guess to base from a given table
    # table of lidar name, xyz translation [m], rpy rotation [deg]
    lidar_1: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    lidar_2: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    lidar_3: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]

    # path for point clouds in pcd format. Structure: pcd_directory/<lidar_name>.pcd
    pcd_directory: /../data/demo/

    # logging output
    results_file: results.txt

    # calibration results are stored here (path relative to the dir with multi_lidar_calibrator.py)
    default_output_dir: /../output/
